x-pack/plugins/alerting/server/saved_objects/schemas/raw_rule/v1.ts

```diff
@@ -194,7 +194,7 @@ const rawRuleAlertsFilterSchema = schema.object({
 
 const rawRuleActionSchema = schema.object({
   uuid: schema.maybe(schema.string()),
-  group: schema.string(),
+  group: schema.maybe(schema.string()),
```

Comment:
make sense!, However, I am wondering if we should do something more fancy and have one type for system action and a default one which can be derived from a new attribute `type: 'system' | 'default' = 'default'`

Reply:
kbn-schema is VERY temperamental about deriving optional types based on attributes, even more so than Typescript itself. I started out trying to do something like this, but the amount of downstream type changes it ended up requiring were massive.

Reply:
@cnasikas what do you think?

Reply:
@cnasikas Yes that's correct. I think let's keep it as a single schema in this case.